bin/test-workflow/start

```diff
@@ -23,12 +23,20 @@ Usage: ./start [options]:
                                 - Defaults to 'gke'
                                 - Supports 'jenkins'
                               All other selections are rejected
+    -c, --ci-apps             Run the E2E workflow against the subset of apps tested in CI
```

Comment:
What happens if both `--ci-apps` and `--apps` arguments are used? Looks like whichever one is added last takes precedence, maybe?

Reply:
I think that makes sense, and is more deterministic.